test/rules/import-blacklist/test.ts.lint

```diff
@@ -2,17 +2,46 @@ import { Observable } from 'rxjs';
                             ~~~~ [0]
 import { Observable } from 'rxjs/Observable';
 
-import forOwn = require('lodash');
-                         ~~~~~~ [0]
-import forOwn = require('lodash/forOwn');
+import rxjs = require('rxjs');
+                       ~~~~ [0]
+import Observable = require('rxjs/Observable');
+
+import * as rxjs from "rxjs";
+                       ~~~~ [0]
 
 // non-static imports cannot be checked
 import {Observable} from rxjs;
 import forOwn = require(lodash);
 
 import * as notBlacklisted from "not-blacklisted";
 
-export * from 'lodash';
-               ~~~~~~ [0]
+export * from 'rxjs';
+               ~~~~ [0]
+
+export { default } from "dummy";
+       ~~~~~~~~~~~ [3]
+
+export * from "lodash";
+               ~~~~~~ [1]
+
+import * as lodash from "lodash";
+                         ~~~~~~ [1]
+import { difference } from "lodash";
+
+import { pull, difference } from "lodash";
+       ~~~~~~~~~~~~~~~~~~~~ [2]
```

Comment:
is there a way to not underline `difference`?

Reply:
I assume so, but I really don't think it's worth it — the error message already specifies which is the problematic import.